tools/c7n_terraform/tests_terraform/data/aws-s3-bucket/s3.tf

```diff
@@ -0,0 +1,24 @@
+resource "aws_s3_bucket" "b" {
+  bucket = var.mybucket
+  acl    = "public-read"
+  policy = "${file("policy.json")}"
```

Comment:
should this file exist?

Reply:
this seems valid re these files being valid terraform modules.